components/ui/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,139 @@
+import { cva, VariantProps } from "class-variance-authority";
+import { Menu, Transition } from "@headlessui/react";
+import { Fragment } from "react";
+import { ChevronDownIcon } from "@heroicons/react/20/solid";
+
+const dropDownStyles = cva("", {
+  variants: {
+    items: {
+      primary: "bg-blue-200",
+      secondary: "bg-green-200",
```

Comment:
When I click the secondary variant I don't see the background change.

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/25281692/199792789-efd4d2b1-52b7-407f-878f-51a4482701e3.png">

Reply:
Yea the primary and secondary are for changing the color of the dropdown items. So if you open the dropdown you can change between primary and secondary colors when hovering over the items